server/catgenome/src/main/java/com/epam/catgenome/util/motif/MotifSearchIterator.java

```diff
@@ -46,6 +47,8 @@
     private static final byte LOWERCASE_T = 't';
     private static final byte LOWERCASE_N = 'n';
 
+    private static int maxSizeResultLimit = Integer.MAX_VALUE;
```

Comment:
we need a way to configure this value, lets move it to manager and make it @Value, moreover Integer,MAX_VALUE is useless because list already can have size no more than MAX_VALUE